tests/helpers.py

```diff
@@ -236,7 +250,7 @@ def create_pvc(sc_name, pvc_name=None):
     pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
     pvc_data['spec']['storageClassName'] = sc_name
     return create_resource(
-        desired_status=constants.STATUS_BOUND, **pvc_data
+        desired_status=constants.STATUS_BOUND, **pvc_data, wait=wait
```

Comment:
this should be changed right? to PVC()

Reply:
Should be done as part of https://github.com/red-hat-storage/ocs-ci/pull/357